study/src/test/java/di/stage4/annotations/ClassPathScanner.java

```diff
@@ -1,10 +1,13 @@
 package di.stage4.annotations;
 
 import java.util.Set;
+import org.reflections.Reflections;
+import org.reflections.scanners.Scanners;
 
 public class ClassPathScanner {
 
-    public static Set<Class<?>> getAllClassesInPackage(final String packageName) {
-        return null;
+    public static Set<Class<?>> getAllClassesInPackage(String packageName) {
+        Reflections reflections = new Reflections(packageName, Scanners.SubTypes.filterResultsBy(filter -> true));
```

Comment:
[질문]

```suggestion
        Reflections reflections = new Reflections(packageName, Scanners.SubTypes.filterResultsBy(filter -> true));
```

오... 여기서 Scanners.SubTypes.filterResultsBy 와 getSubTypesOf가 어떤 역할을 하는지 궁금해요 😮

Reply:
아하 기존에는 객체 자체만 가져왔다면 부모 클래스와 인터페이스도 스캔 대상에 포함하게 해주는 것이군요 😄 
덕분에 많이 배워가네요 제리 🐹 친절히 설명해주어서 감사합니다!